nautilus 2.26+ uses GIO instead of GnomeVFS

--- origsrc/nautilus-python/configure.in	2009-05-03 04:04:03.000000000 -0500
+++ src/nautilus-python/configure.in	2009-05-03 04:09:23.343762100 -0500
@@ -39,6 +39,7 @@ AM_CHECK_PYTHON_LIBS(,[AC_MSG_ERROR(coul
 PKG_CHECK_MODULES(NAUTILUS_PYTHON, [pygtk-2.0 >= $PYGTK_REQUIRED
                                    gnome-python-2.0 >= $GNOME_PYTHON_REQUIRED
                                    libnautilus-extension >= $NAUTILUS_REQUIRED
+                                   gnome-vfs-2.0
                                    eel-2.0 >= $EEL_REQUIRED])
 
 AC_MSG_CHECKING(for pygtk defs)
